corehq/apps/reports/standard/users/reports.py

```diff
@@ -179,15 +180,39 @@ def _html_list(changes, unstyled=True):
 
     properties = UserHistoryReport.get_primary_properties(domain)
     properties.pop("user_data", None)
-    primary_changes = {
-        properties.get(key, key): value for key, value in changes.items()
-        if key in properties
-    }
-    more_count = len(changes) - len(primary_changes)
+    primary_changes = {}
+
+    for key, value in changes.items():
+        if key in properties:
+            if key == 'commcare_location_id':
+                primary_changes[_("primary location (mobile users only)")] = cached_location_id_to_name(value)
```

Comment:
Is the `_` intentional?

Reply:
Yeah, I was assuming based on get_primary_properties that the translation was helpful/needed, although it's not being used for other properties like "user_data"  and "commcare_location_id". Is it better to just leave out?

Reply:
It was definitely the inconsistency that caught my eye, either everything or nothing should be translated. Given that this dict is purely used for the UI, I think using translated strings would be fine and good. If these keys were persistently saved anywhere, translating them probably wouldn't work out.

@mkangia pinging you since you've thought a lot about translations in this report.

Reply:
> Given that this dict is purely used for the UI, I think using translated strings would be fine and good

+1